node/test/lib/relay_network.js

```diff
@@ -46,13 +46,18 @@ function RelayNetwork(options) {
     self.serviceNames = options.serviceNames || ['alice', 'bob', 'charlie'];
     self.kValue = options.kValue || 2;
     self.createCircuits = options.createCircuits || noop;
-    self.clusterOptions = options.cluster || {};
```

Comment:
Unless RelayNetwork is already used outside of TChannel, all call sites are verified.